bitnami/elasticsearch/README.md

```diff
@@ -700,7 +700,7 @@ kibana:
         # Instruct kibana to connect to elastic over https
         enabled: true
         # Bit of a catch 22, as you will need to know the name upfront of your release
-        existingSecret: RELEASENAME-elasticsearch-coordinating-only-crt
+        existingSecret: RELEASENAME-elasticsearch-coordinating-crt # or just 'elasticsearch-coordinating-crt' if the release name happens to be 'elasticsearch'
```

Comment:
It seems this depends on which kind of Elastic nodes are actually enabled for specific deployment. If e.g. `coordinating` is disabled, then this secret would not exist and one would need to reference `data` or `master` certs. However, since by default `coordinating` nodes are enabled, I think this is OK.

I could possibly add additional note clarifying the secret naming, but it would clutter the README, so not sure if that's worth it (since this is just an example anyway).